src/NuGetGallery.Services/Configuration/FeatureFlagService.cs

```diff
@@ -39,6 +39,7 @@ public class FeatureFlagService : IFeatureFlagService
         private const string UsabillaOnEveryPageFeatureName = GalleryPrefix + "UsabillaEveryPage";
         private const string PackageRenamesFeatureName = GalleryPrefix + "PackageRenames";
         private const string EmbeddedReadmeFlightName = GalleryPrefix + "EmbeddedReadmes";
+        private const string LicenceMdRenderingFlightName = GalleryPrefix + "LicenceMdRendering";
```

Comment:
It looks like the correct spelling should be "license" instead of "licence" ("license" is the noun, "licence" is the verb). 

```suggestion
        private const string LicenseMdRenderingFlightName = GalleryPrefix + "LicenseMdRendering";
```